CONTRIBUTING.md

```diff
@@ -102,6 +102,11 @@ docker build -t ovirt-site .
 docker run -v $(pwd):/srv/site -p 4000:4000 -p 35729:35729 -ti ovirt-site
 ```
 
+Please note that on SELinux enforcing systems the last command should be:
```

Comment:
LGTM, just 1 suggestion:
If I understand correctly the below command comes **instead** of the `docker run -v $(pwd):/srv/site -p 4000:4000 -p 35729:35729 -ti ovirt-site` above, so maybe better to emphasize that with something like:
`...the last command should be:`
-->
`...the last command above should change to:`

Reply:
thanks, updated.